### PR TITLE
EHPR-76: Add basic deployment pipeline

### PR DESCRIPTION
**Pull request by fwpushan**
Add basic pipeline to deploy application on aws dev env.

- [x] Adhoc deployment with pipeline execution 
- [x] Env specific isolation 
- [ ] Pipeline execution for PR merge.

### CODE REVIEW

**Review by arranfw — APPROVED** [review]
Looks right

File: Makefile [hunk 1]
```diff
@@ -120,6 +120,7 @@ build-api:
 
 build-web:
 	@echo "++\n***** Building Web for AWS\n++"
+	@yarn workspace @ehpr/web build
```
Comment (arranfw) [inline Makefile:123]: Export already runs build doesn't it?

Comment (fwpushan) [reply Makefile:123]: It was crashing because not able to locate build dir

Comment (fwpushan) [reply Makefile:123]: It is due to build common

Comment (fwpushan) [reply Makefile:123]: yarn workspace @ehpr/web build => a. build dependencies + b. web

Comment (arranfw) [reply Makefile:123]: 🤔 Can we remove the extra build from the export step then? This looks like it'll build web twice

Comment (fwpushan) [reply Makefile:123]: `yarn workspace @ehpr/web build` only builds dependencies for web and web. That a improvement over build-common.

**Review by dbayly-freshworks — APPROVED** [review]
Looks good to me!

**Pull request merged**